server/flyway/sql/V66__add_admin_role_to_fom.sql

```diff
@@ -0,0 +1,30 @@
+-- Add a CLIENT_SUSPEND role to Forest Client DEV and TEST
+WITH application_ids AS (
+    SELECT
+        application_id,
+        application_name
+    FROM
+        app_fam.fam_application
+    WHERE
+        application_name IN ('FOM_DEV', 'FOM_TEST')
+)
+INSERT INTO
+    app_fam.fam_role (
+        role_name,
+        role_purpose,
+        application_id,
+        role_type_code,
+        create_user,
+        create_date,
+        display_name
+    )
+SELECT
+    'FOM_ADMIN',
+    'Manage FOM analytics summary data.',
```

Comment:
Is this a good desccription?

Reply:
I updated to "Manage and access FOM analytics summary data, with additional permissions to oversee and support administrative tasks." But not sure if this is good, I asked Olga in the ticket.